framework/src/postprocessors/ExtremeValueBase.C

```diff
@@ -71,20 +73,10 @@ template <class T>
 void
 ExtremeValueBase<T>::finalize()
 {
-  if (_type == ExtremeType::MAX)
-  {
-    if (_use_proxy)
-      this->gatherProxyValueMax(_proxy_value.first, _proxy_value.second);
-    else
-      this->gatherMax(_proxy_value.second);
-  }
```

Comment:
if we're going to do this simplification then we can just remove `_use_proxy`. I don't think there's much of a performance penalty here so I'm fine with this